### DISCUSSION TITLE
i2c-1, i2c-0

### DISCUSSION THREAD

**Discussion by pkondela** [jpcornil-git/HA-mcp23017#32] [Ideas]
Hello, can i2c-1 and i2c-0 be used at the same time?  i have multiple devices and this would really help.
Thanks

**Comment by jpcornil-git** [comment]
Hi,
No, it is not but you can add multiple devices to the same bus (up to 128) as long as they have different adresses
Cheers,
jpc

**Comment by pkondela** [comment]
hi, thank you very much, but 128 devices is really not enough for me 😉

**Comment by jpcornil-git** [comment]
Up to 128 devices but max. 8 mcp23017 (3b address) or 128 ports ... not too bad already :-)

**Comment by jpcornil-git** [comment]
Feature request has been developped and checked-in ([f7971e1](https://github.com/jpcornil-git/HA-mcp23017/commit/f7971e171420ec9fbd7d879abbdfbc76fa2aff2b))
